core/Updates/4.0.0-b1.php

```diff
@@ -269,6 +246,9 @@ public function doUpdate(Updater $updater)
 
         $updater->executeMigrations(__FILE__, $this->getMigrations($updater));
 
+        // just in case it was not executed manually by a user...
+        MigrateTokenAuths::migrate();
```

Comment:
Is this necessary? Since the migration is added above, if the user clicks 'Update', it will definitely run right? If so, this would be redundant. Or is there some way for the user to opt out of that specific migration running?

Reply:
true, it's not needed .will remove it now